### PR TITLE
MSEARCH-284 Missing 's' in query property for holdingsIdentifiers

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
### Approach
Rename 
1. holdingIdentifiers    -> holdingsIdentifiers
2. holdingPublicNotes -> holdingsPublicNotes
3. holdingTags             -> holdingsTags
4. itemsNormalizedCallNumbers -> itemNormalizedCallNumbers
5. itemsFullCallNumbers -> itemFullCallNumbers

### Learning
[MSEARCH-284](https://issues.folio.org/browse/MSEARCH-284)

### CODE REVIEW

**Review by pfilippov-epam — APPROVED** [review]
Looks good, but this change will require to reindex after merging it

**Comment by psmagin** [comment]
@Shans-Kaluhin, could you please also change `itemsFullCallNumbers` and `itemsNormalizedCallNumbers` fields that was mentioned in comment:  
https://issues.folio.org/browse/MSEARCH-284?focusedCommentId=122978&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-122978

File: src/main/resources/model/instance.json [hunk 1]
```diff
@@ -484,7 +484,12 @@
     "holdingPublicNotes": {
       "type": "search",
       "index": "multilang",
-      "processor": "holdingPublicNotesProcessor"
+      "processor": "holdingsPublicNotesProcessor"
+    },
+    "holdingsPublicNotes": {
```
Comment (pfilippov-epam) [inline src/main/resources/model/instance.json:489]: there is no need to duplicate the search fields, old fields can be called with the new values using the 
property `inventorySearchTypes`

working example:
```
    "holdingPublicNotes": {
      "type": "search",
      "index": "multilang",
      "inventorySearchTypes": [ "holdingsPublicNotes" ],
      "processor": "holdingsPublicNotesProcessor"
    }
```

Comment (Shans-Kaluhin) [reply src/main/resources/model/instance.json]: Thanks Pavel!

**Review by psmagin — APPROVED** [review]

**Review by viacheslavkol — APPROVED** [review]

**Pull request merged**